src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPopupPanel.cs

```diff
@@ -35,6 +38,30 @@ public FlyoutBasePopupPanel(FlyoutBase flyout) : base(flyout._popup)
 		internal override FlyoutBase Flyout => _flyout;
 
 		protected override int PopupPlacementTargetMargin => 5;
+
+		protected override void OnPointerPressed(object sender, PointerRoutedEventArgs args)
+		{
+			base.OnPointerPressed(sender, args);
+
+			// Make sure we are the original source.  We do not want to handle PointerPressed on the Popup itself.
+			if (args.OriginalSource == this)
+			{
+				if (Flyout.OverlayInputPassThroughElement is UIElement passThroughElement)
+				{
+					var (elementToBeHit, _) = VisualTreeHelper.SearchDownForTopMostElementAt(
+						args.GetCurrentPoint(null).Position,
+						passThroughElement.XamlRoot.VisualTree.RootElement,
+						VisualTreeHelper.DefaultGetTestability,
+						childrenFilter: elements => elements.Where(e => e != this));
+					
+					var eventArgs = new PointerRoutedEventArgs(
+						new PointerEventArgs(args.GetCurrentPoint(null), args.KeyModifiers),
+						args.OriginalSource as UIElement);
+					
+					elementToBeHit.OnPointerDown(eventArgs);
```

Comment:
I'm not sure if this will exactly match the behaviour on Windows. I haven't tested this yet, but I'm not sure of the order of events in the case of having event handlers with HandledEventsToo. Should the first event completely propagate to the top and only then trigger the new event, or should the new event be triggered within the first event propagation cycle (like I've done here)?